apps/docs/src/app/core/component-docs/card/card-docs.component.html

```diff
@@ -145,3 +145,15 @@
     <fd-card-link-list-example></fd-card-link-list-example>
 </component-example>
 <code-example [exampleFiles]="linkListCard"></code-example>
+
+<description>Image type Link list card</description>
```

Comment:
Could you add `fd-docs-section-title` component, instead of `description`?

Reply:
using `fd-docs-section-title` and `description` both now.